docs/how-to-run.adoc

```diff
@@ -266,26 +266,44 @@ last = "192.168.1.29"
 
 This is a range of IP addresses on your external network that Omicron can assign to externally-facing services (like DNS and the API).  You'll need to change these if you've picked different addresses for your external network.  See <<_external_networking>> above for more on this.
 
+You will also need to update route information if your `$GATEWAY_IP` differs from the default:
+
 [source,toml]
 ----
 # Configuration to bring up boundary services and make Nexus reachable from the
 # outside.  This block assumes that you're following option (2) above: putting
 # your Oxide system on an existing network that you control.
 [rack_network_config]
-# The gateway for the external network
-gateway_ip = "192.168.1.199"
+rack_subnet = "fd00:1122:3344:01::/56"
 # A range of IP addresses used by Boundary Services on the network.  In a real
 # system, these would be addresses of the uplink ports on the Sidecar.  With
 # softnpu, only one address is used.
 infra_ip_first = "192.168.1.30"
 infra_ip_last = "192.168.1.30"
-# Name of the port.  This should always be "qsfp0" when using softnpu.
-uplink_port = "qsfp0"
-uplink_port_speed = "40G"
-uplink_port_fec="none"
+
+# Configurations for BGP routers to run on the scrimlets.
```

Comment:
For this value and the ones below, it's not clear to me how I know what values to set here, particularly for a clustero n a home network, where I assume BGP is not a factor. Could we add some color on that?

Reply:
Thanks! I've extended these descriptions after quickly following `common::api::internal::shared::{RackNetworkRackNetworkConfigServices, BgpConfig}` around sled-agent. I haven't had any time to play around with BGP as yet, maybe @rcgoodfellow can verify this makes sense?

Reply:
Yeah, this makes sense. I'll add an update to the [BGP Docs PR](https://github.com/oxidecomputer/docs/pull/216) that updates the [network preparations RSS docs](https://docs-git-bgp-guide-oxidecomputer.vercel.app/guides/system/network-preparations#_complete_configuration) in addition to the [BGP Guide](https://docs-git-bgp-guide-oxidecomputer.vercel.app/guides/operator/configuring-bgp). Then we can just provide a link to the user/operator facing docs here.